policyengine_us/parameters/gov/states/vt/tax/income/credits/elderly_or_disabled/match.yaml

```diff
@@ -0,0 +1,14 @@
+description: Vermont matches this fraction of the federal credit for elderly or the disabled. 
+values:
+  2021-01-01: 0.24
+metadata:
+  unit: /1
+  period: year
+  label: Vermont elderly or the disabled tax credit match
+  reference:
+    - title: 32 V.S.A. §5822. Tax on income of individuals, estates, and trusts (d)(1)
+      href: https://legislature.vermont.gov/statutes/section/32/151/05828b
```

Comment:
I am not sure if this link has a d 1 part

Reply:
Updated